### PR TITLE
Tutorial S2 map overhaul

### PR DESCRIPTION
**Pull request by Earth-Cake**
Some players have commented on the tutorial S2 being too hard. This PR aims to make it easier by redoing the map.

### CODE REVIEW

File: data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg [hunk 1]
```diff
@@ -1095,7 +1102,7 @@ Please report the bug."
 
         [message]
             speaker=Galdrad
-            message= _ "Higher level units are powerful, but not invulnerable. Goodbye, $unit.name|."
+            message= _ "Higher level units are powerful, but not invulnerable. We'll miss you, $unit.name|."
```
Comment (stevecotton) [inline data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg]: Needs a typographical-style apostrophe.

File: data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg [hunk 2]
```diff
@@ -1117,9 +1124,11 @@ Please report the bug."
         name=moveto
         [filter]
             side=1
-            x=1-20
-            y=18-21
         [/filter]
+        [filter_location]
+            x,y=38,5
+            radius=5
+        [/filter_location]
```
Comment (stevecotton) [inline data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg]: I couldn't get this one to trigger

Comment (Earth-Cake) [reply data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg]: Strangely enough, it works perfectly fine for me.

File: data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg [hunk 3]
```diff
@@ -511,14 +519,14 @@ A full list of abilities and weapons specials, along with traits, may be found i
                 # 9,12 for a fight, in which case let's show the sanity check
                 # failure message anyway).
                 [have_unit]
-                    x=8-10
-                    y=8-12
+                    x=10-14
+                    y=9-12
```
[the remark this answers is not in the document]
Comment (Earth-Cake) [reply data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg]: Sure, I overlooked that line.

File: data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg (hunk 2 above)
Comment (stevecotton) [reply data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg]: I retried, and it seems broken but in a different way - it triggers on the very first move by side 1, when Konrad/Li'sar steps off their keep.

Comment (Earth-Cake) [reply data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg]: Huh, happens to me too.

Comment (Earth-Cake) [reply data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg]: This should fix it.

**Comment by stevecotton** [comment]
@Earth-Cake and @nemaara, I think we should backport this, b0aed28de1a0df209140bf86a9fc8bc3ab981ea4 and ce3ac1e33d0c70e8d8d3d799d6a8005d49a128c8 to 1.14. I'm about to do some testing for PR #4418 so could do the backport too.

**Comment by nemaara** [comment]
Didn't know this kind of change could/should be backported, but sure.

**Pull request merged**